Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -376,23 +376,33 @@ public class PBXProjGenerator {
 
     func sortGroups(group: PBXGroup) {
         // sort children
-        let children = group.children
-            .sorted { child1, child2 in
-                let sortOrder1 = child1.getSortOrder(groupSortPosition: project.options.groupSortPosition)
-                let sortOrder2 = child2.getSortOrder(groupSortPosition: project.options.groupSortPosition)
-
-                if sortOrder1 != sortOrder2 {
-                    return sortOrder1 < sortOrder2
-                } else {
-                    if child1.nameOrPath != child2.nameOrPath {
-                        return child1.nameOrPath.localizedStandardCompare(child2.nameOrPath) == .orderedAscending
+        if let groupOrder = project.options.groupsOrder.first(where: { group.nameOrPath == $0.pattern || group.nameOrPath.isMatch(to: $0.pattern) }) {
+            let files = group.children.filter { $0 is PBXFileReference }
```

Comment:
Is there any reason files couldn't be sorted? What if all the sorts were sorted first and then everything else came after? I guess there could be an option to put everything else first as well

Reply:
You know, I just forgot to sort files. I divided files and groups and sorted groups but forgot to sort files. So I can sort files and groups and after this merge it by `fileSortPosition` option.